bcipy/gui/gui_main.py

```diff
@@ -196,6 +196,58 @@ def widgets(self) -> List[QWidget]:
         return [self.label_widget, self.help_tip_widget, self.control, self]
 
 
+class IntegerInput(FormInput):
+    """FormInput to select a integer value using a spinbox for selection. Help text is not
+    displayed for spinbox items.
+
+    Parameters:
+    ----------
+        label - form label.
+        value - initial value.
+    """
+
+    def __init__(self, **kwargs):
+        super(IntegerInput, self).__init__(**kwargs)
+
+    def init_control(self, value):
+        """Override FormInput to create a spinbox."""
+        spin_box = QSpinBox()
+        spin_box.setMaximum(100000)
+        return spin_box
+
+    def value(self) -> str:
+        """Override FormInput to return an integer value."""
+        if self.control:
+            return int(self.control.text())
+        return None
```

Comment:
See comment below regarding the return type. The signature here says it returns a `str`, but the code may return an `int`.